tmt/utils.py

```diff
@@ -1780,3 +1811,78 @@ def get_distgit_handler(remotes=None, usage_name=None):
 def get_distgit_handler_names():
     """ All known distgit handlers """
     return [i.usage_name for i in DistGitHandler.__subclasses__()]
+
+
+class updatable_message(contextlib.AbstractContextManager):
```

Comment:
Shouldn't class name start with upper case?

Reply:
I'd say being used as a context manager pushes the first letter into the lowercase territory. Similar to `open`, `socket.create_connection`, `urllib.request.urlopen` (although those are not classes, just context managers), (very soft) naming convention seems to be in favor of context managers and decorators following function naming convention.

And `with UpdatableMessage('state', indent_level=self._level()) as progress_message:` looks just weird :)

Reply:
Yeah, for context manager it makes sense to have it lower case.